google-chart.html

```diff
@@ -197,6 +199,25 @@
           this.options = {};
           this.rows = [];
           this.dataTable = null;
+
+          var head = document.querySelector('head');
+          var element = this;
+          var observer = new MutationObserver(function(mutations) {
+              mutations.forEach(function(mutation) {
+                for (var i = 0; i < mutation.addedNodes.length; i++) {
+                  var newNode = mutation.addedNodes[i];
+                  if (newNode.rel && newNode.rel == 'stylesheet') {
+                    //Add the stylesheet as an @import
+                    var cssImport = document.createElement('style');
+                    cssImport.innerHTML = "@import '" + newNode.href + "';";
+                    element.shadowRoot.appendChild(cssImport);
```

Comment:
Doesn't this just append *any* dynamically added stylesheet to the element's shadow DOM? I don't think this is a viable option unless we can reliably identify gviz stylesheets only.

Also, I think gviz only adds the stylesheets once, for the first chart. This means the mutation observer for `<google-charts>` added after the gviz style are added to the `<head>` won't pick up these styles.